src/util/userMetricsCookie.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * This is a helper function to update cookies values for user metrics.
+ * it is going to be used for Optimizely A/B experiment.
+ * @param cookieStore
+ * @param loans
+ * @param receipt
+ */
+export default function handleUserMetricsCookies(cookieStore, loans, receipt) {
+	const depositTotal = receipt?.totals?.depositTotals?.depositTotal;
+	const hasLentBefore = loans?.length > 0;
+	const hasDepositBefore = parseFloat(depositTotal) > 0;
+
+	if (hasLentBefore || hasDepositBefore) {
+		cookieStore.remove('kvu_lb');
```

Comment:
Now that there is this dedicated file, recommend creating 2 new constants for these cookie names. Using a shared variable is generally preferred over magic strings.

For example, in this method at the top: `const lentBeforeCookieKey = 'kvu_lb';`